webpack.config.js

```diff
@@ -13,8 +13,8 @@ const BundleAnalyzerPlugin =
   require("webpack-bundle-analyzer").BundleAnalyzerPlugin;
 
 // default value is set in the code where it is used
-if (!process.env.WALLET_CREATE_URL) {
-  process.env.WALLET_CREATE_URL = ""; // env variables are passed as string. empty strings are still falsy
+if (!process.env.WALLET_ROOT_URL) {
```

Comment:
Update build.yml and all occurances of WALLET_CREATE_URL (docs + playwright.config)